dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/hibernate/JsonUserType.java

```diff
@@ -0,0 +1,206 @@
+package org.hisp.dhis.hibernate;
+
+/*
+ * Copyright (c) 2004-2017, University of Oslo
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
```

Comment:
Have a look at "JsonBinaryType.java" in dhis-support-hibernate. Please use this type. We cannot have multiple JSON types in the system.

Reply:
By the time we did this development there was no support for json types in Dhis2 so we implemented the support in a different way. We have just changed so we use JsonBinaryType...